less/site.less

```diff
@@ -203,4 +203,13 @@ a {
 
 .no-padding {
   padding: 0;
+}
+
+.glyphicon {
+  top: 2px;
+}
+
+.bold {
```

Comment:
I'm not always picky on these, but could a more semantic name be used (e.g. `player-name`)? It's narrowly scoped enough to be used as a general "make this bold" class, but the white color could be a surprise in other usages.

Reply:
That’s actually the intent. The nature of the site wide foreground is that it looks suboptimal as bold. So everywhere that’s bold I want to look like this

Reply:
👍